docs/tutorials/adding-a-support-desk-button.md

```diff
@@ -0,0 +1,243 @@
+
+## Integration Approach
+
+### Adding the Support Button to the Admin Bar
+
+In ApostropheCMS, you can add a support desk button to the admin bar from the `init` method of any module.
+
+The code is straightforward, using the `self.apos.adminBar.add()` method to register the new button:
+
+<AposCodeBlock>
+
+```javascript
+export default {
+  init(self) {
+    // Add support desk button to the admin bar
+    self.apos.adminBar.add(
+      'support-desk',           // Unique identifier
+      'Get Support',            // Button label
+      false,                    // Available to all users with admin access
+      {
+        contextUtility: true,   // Places button in the right side utility area
+        icon: 'question-circle', // Icon for the button
+        tooltip: 'myProject:supportDesk' // Namespaced tooltip text for translation
+      }
+    );
+  }
+};
+```
+  <template v-slot:caption>
+    modules/asset/index.js
+  </template>
+</AposCodeBlock>
+
+In this example we have added the button to the context menu, but you may choose to add it as a main navigation item or the user drop-down menu. See the [reference documentation for other placement options](/reference/modules/admin-bar.html#add-name-label-permission-options).
+
+This single method call is all you need to add the button to the admin bar. The first parameter is a unique identifier that you'll reference in your event handler. The second parameter is the visible label (ideally using a translation string in production). The third parameter controls permissions, where `false` makes it available to all users with admin access. The final parameter is an options object that determines the button's placement and appearance.
+
+### Handling the Button Click Event
+
+Once you've added the support button to the admin bar, you need to create a browser-side component to handle clicks. This is where you'll connect your admin bar button to the third-party support desk widget's API.
+
+In your project's UI JavaScript file, add an event listener for the admin bar button click and trigger the appropriate support widget:
+
+<AposCodeBlock>
+
+```javascript
+export default () => {
+  // Listen for admin bar button clicks once ApostropheCMS is loaded
+  apos.bus.$on('ready', () => {
+    apos.bus.$on('admin-menu-click', (item) => {
+      // Check if our support desk button was clicked
+      if (item === 'support-desk') {
+        openSupportWidget();
+      }
+    });
+  });
+
+  function openSupportWidget() {
+    // Call the appropriate API based on which support desk you're using
+    // The exact method will vary by provider - check their documentation
```

Comment:
Any reason not to go with a functioning example for one of them?

Reply:
Added.